app/models/concerns/profile_searching.rb

```diff
@@ -148,6 +149,33 @@ def filter_by_policy(_filter, _operator, policy_or_profile_id)
         conditions: "profiles.id IN (#{any_child_profile.to_sql})"
       }
     end
+
+    def filter_by_policy_and_profile_names(_filter, operator, value)
+      values = preprocess_parameters(operator, value)
```

Comment:
As I wrote above this is not needed at all :see_no_evil:

Reply:
Referring to this comment: 
"According to [this](https://github.com/RedHatInsights/compliance-backend/blob/master/config/initializers/override_scoped_search_escaping.rb), the parameters are already preprocessed when you're getting them. The _ symbols are used to swap the % for like and unlike already. You don't really need to do any additional preprocessing."

The value that's passed to the ext_method `filter_by_policy_and_profile_names` isn't being updated by the preprocess_parameters [here](https://github.com/RedHatInsights/compliance-backend/blob/master/config/initializers/override_scoped_search_escaping.rb). The original value is always passed to the ext_method. I think it's just how the `ext_method`'s work or maybe it's a bug with search_scope

Reply:
Maybe, but then you can just call the existing method :thinking:

Reply:
I thought about that too but there are two parameters that need to be passed into that one,`field` and `block`, and I'm not sure how to pass the correct values for those.. But I can try to figure it out!

Reply:
I wasn't able to call the `preprocess_parameters` method [here](https://github.com/RedHatInsights/compliance-backend/blob/master/config/initializers/override_scoped_search_escaping.rb) because the `field` param that's needed isn't being passed to the ext_method
The `field` param is an object of the class [ScopedSearch::Definition::Field](https://github.com/wvanbergen/scoped_search/blob/master/lib/scoped_search/definition.rb) and I'm not sure if it's possible accessing this object from the `filter_by_policy_and_profile_names` :confused:

Reply:
If I understand it [here](https://github.com/wvanbergen/scoped_search/blob/master/lib/scoped_search/query_builder.rb#L219) correctly, `this` might contain the `field` that you need. I'll give it a try and get back to you.

Reply:
Yeah, maybe I'm going to try now too

Reply:
My comment didn't post on Friday 😕

I figured out how to get the value of `field`. Based on [this](https://github.com/wvanbergen/scoped_search/blob/10d118690b6ba5a4941ac17e3d9157ddc817592f/lib/scoped_search/query_builder.rb#L425) `definition.klass` is `self` in the ext_method so I was able to get the `field` value by doing `self.scoped_search_definition.field_by_name('name')`
But now I need the ScopedSearch::QueryBuilder instance to call the `preprocess_parameters` method. Still trying to figure out if this is possible..

Reply:
I've looked through the scoped_search code and it doesn't look like there's any way of getting the `ScopedSearch::QueryBuilder::PostgreSQLAdapter` instance that's needed to call the `preprocess_parameters` method